polaris-migrator/src/migrations/react-breadcrumbs-migrate-from-array/tests/react-breadcrumbs-migrate-from-array.input.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {Page} from '@shopify/polaris';
+
+export function App() {
+  return (
+    <>
+      <Page breadcrumbs={[{url: '/testing', content: 'Breadcrumb'}]}>
+        hello
+      </Page>
+      <Page
+        breadcrumbs={[
+          {url: '/testing', content: 'Breadcrumb'},
+          {url: '/testing2', content: 'Breadcrumb2'},
+        ]}
+      >
+        hello
+      </Page>
+      <Page breadcrumbs={{url: '/testing', content: 'Breadcrumb'}}>
+        testing
+      </Page>
+      <Page breadcrumbs={[]}>testing</Page>
```

Comment:
Could we add a test case here for handling dynamic props when a variable is used?

```tsx
const breadcrumbs = [{ ... }];

<Page breadcrumbs={breadcrumbs}>testing</Page>
```

Inserting a comment in this case would be totally fine in my opinion. Example output would be the following

```tsx
{/* polaris-migrator: Unable to migrate the following expression. Please upgrade manually. */}
<Page breadcrumbs={breadcrumbs}>testing</Page>
```

You can do this pretty easily using a utility [as done here in this example](https://github.com/Shopify/polaris/blob/main/polaris-migrator/src/migrations/react-replace-text-components/steps/replace-text-style.ts#L83)!

Reply:
Sam is suggesting you skip these occurrences and put a comment above the line of code. I think we do this with the `Text` component migration as well.